source/core/TDCodecVia.cpp

```diff
@@ -1059,13 +1059,14 @@ Int32 TDViaParser::ParseData(TypeRef type, void* pData)
         case kEncoding_UInt:
         case kEncoding_S2CInt:
             {
-                Boolean is64Bit = type->IsA(&TypeCommon::TypeInt64) || type->IsA(&TypeCommon::TypeUInt64);
+                Boolean is64Bit = type->IsInteger64();
                 IntMax value = 0;
                 Boolean overflow = false;
                 Utf8Char sign = 0;
                 _string.EatWhiteSpaces();
+                Boolean ateBeginningQuote = false;
```

Comment:
Maybe we can choose a name that better matches the intent like `isNumericString`